src/main/webapp/app/entities/quiz-exercise/quiz-exercise-detail.component.ts

```diff
@@ -16,8 +16,10 @@ import { AnswerOption } from '../../entities/answer-option';
 import { Option, Duration } from './quiz-exercise-interfaces';
 import { NgbDateStruct, NgbDate, NgbTimeStruct } from '@ng-bootstrap/ng-bootstrap';
 import * as moment from 'moment';
+import { ComponentCanDeactivate } from '../../shared';
 import { Moment } from 'moment';
 import { JhiAlertService } from 'ng-jhipster';
+import { Observable } from '../../../../../../node_modules/rxjs/Observable';
 
```

Comment:
please fix this import to avoid so many ../../
Have a look how other files import rxjs Observable

Reply:
Done and updated - @krusche please review